### PR TITLE
[Review App] Scale images by area in artwork shelves

### PR DESCRIPTION
**Pull request by dzucconi**
This converts the artwork shelf to target an area (default 40,000) when coming up with a width to resize to. This allows for a more visually balanced shelf component. Previously we were just constraining everything by a specific width (default 200px), which would lead to large portrait oriented images and small landscape ones (the artwork brick columns still suffer from this problem).

----

Additionally I've gone around and replaced all of the bespoke skeletons with a single placeholder type which should prevent layout shifting for these components.

### CODE REVIEW

File: src/Apps/Consign/Routes/MarketingLanding/Components/SoldRecentlyOnArtsy.tsx [hunk 1]
```diff
@@ -58,29 +52,32 @@ export const SoldRecentlyOnArtsy: React.FC<SoldRecentlyOnArtsyProps> = ({
...
-                  onClick={trackArtworkItemClick(artwork, i)}
-                  // FIXME:
-                  contextModule={ContextModule.artworkRecentlySoldGrid as any}
-                />
+              <ShelfArtworkFragmentContainer
+                key={artwork.internalID}
+                artwork={artwork}
+                hideSaleInfo
+                lazyLoad
+                area={60000}
```
Comment (damassi) [inline src/Apps/Consign/Routes/MarketingLanding/Components/SoldRecentlyOnArtsy.tsx:60]: Hunch is that this is gonna be kinda hard for devs to remember / understand, at least initially. Can you break it down a bit for me just so I understand better? (once i understand completely we might want to come up with a better prop name.. maybe)

Comment (dzucconi) [reply src/Apps/Consign/Routes/MarketingLanding/Components/SoldRecentlyOnArtsy.tsx:60]: I wouldn't have exposed it otherwise.

Comment (damassi) [reply src/Apps/Consign/Routes/MarketingLanding/Components/SoldRecentlyOnArtsy.tsx:60]: Still, might be good to make the name more clear... `maxHorizontalArea` or something. "Area" is so vague! (No strong feelings about it tho other than "whoa, what does area do", which I don't know)

Comment (dzucconi) [reply src/Apps/Consign/Routes/MarketingLanding/Components/SoldRecentlyOnArtsy.tsx:60]: I agree it's weird but it's not a maxHorizontalArea. It's 'how much area do u want this image to occupy?'

Comment (damassi) [reply src/Apps/Consign/Routes/MarketingLanding/Components/SoldRecentlyOnArtsy.tsx:60]: Oh I see. Feels weird no matter how you describe it. I've got no naming ideas other than to suggest there's a good comment on the interface describing its function; `area={60000}` is foreign and indiscernible in terms of function, and whats that super large number? Anywaysss

Comment (dzucconi) [reply src/Apps/Consign/Routes/MarketingLanding/Components/SoldRecentlyOnArtsy.tsx:60]: Yeah like 'totalNumberOfPixelsInImage' isn't a ton better.

Comment (damassi) [reply src/Apps/Consign/Routes/MarketingLanding/Components/SoldRecentlyOnArtsy.tsx:60]: right

**Pull request closed without merging**